### PR TITLE
nabu : Update

### PR DESCRIPTION
**Pull request by Kfkcome**

### CODE REVIEW

File: nabu.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"buildtype": "Monthly",
+			"forum": "https://forum.xda-developers.com/t/rom-android13-crdroid-official-xiaomi-pad-5-nubu-23th-july-2023.4608397/",
+			"gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-T/16-Jul-2023/NikGapps-core-arm64-13-20230716-signed.zip/download",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "",
+			"paypal": "",
+			"telegram": "",
+			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_nabu",
```
Comment (gwolf2u) [inline nabu.json:23]: no 14.0 branch

File: nabu.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"common-dt": "",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_xiaomi_nabu"
```
Comment (gwolf2u) [inline nabu.json:25]: same here

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: nabu.json (hunk 1 above)
Comment (Kfkcome) [reply nabu.json:23]: Sorry,I forget to sync.Now it has.

File: nabu.json (hunk 2 above)
Comment (Kfkcome) [reply nabu.json:25]: Already sync 14.0 branch.

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**